distribution/sender/routes/active-mq-publisher.xml

```diff
@@ -0,0 +1,132 @@
+<routes xmlns="http://camel.apache.org/schema/spring">
+
+    <route id="active-mq-publisher" errorHandlerRef="outBoundErrorHandler">
```

Comment:
Renamed the route id to sender-activemq-publisher

Reply:
Renamed.